flipper-plugin-realm/src/components/DataTable.tsx

```diff
@@ -312,18 +312,22 @@ PropertyType) => {
 
   /** Handling sorting. Is called when the 'state' of the Ant D Table changes, ie. you sort on a column. */
   const handleOnChange = (
-    pagination: TablePaginationConfig,
-    filters: Record<string, Key[] | null>,
+    _pagination: TablePaginationConfig,
+    _filters: Record<string, FilterValue | null>,
```

Comment:
If these aren't being used, you could remove them and pass just the properties you care about from the `Table#onChange` property.  IMO, the onChange should've taken an object for the onChange function arguments, but that's out of our hands.
```
onChange={(_, __, sorter, extra) => handleOnChange(sorter, extra)}
```